src/state/disconnect-toast-mixin.ts

```diff
@@ -1,37 +1,30 @@
 import { Constructor, LitElement } from "lit-element";
 import { HassBaseEl } from "./hass-base-mixin";
-import { HaToast } from "../components/ha-toast";
-import { computeRTL } from "../common/util/compute_rtl";
+import { fireEvent } from "../common/dom/fire_event";
 
 export default (superClass: Constructor<LitElement & HassBaseEl>) =>
   class extends superClass {
-    private _discToast?: HaToast;
-
     protected firstUpdated(changedProps) {
       super.firstUpdated(changedProps);
       // Need to load in advance because when disconnected, can't dynamically load code.
-      import(/* webpackChunkName: "ha-toast" */ "../components/ha-toast");
+      import(/* webpackChunkName: "notification-manager" */ "../managers/notification-manager");
```

Comment:
I don't think that this is needed as the notification manager also needs to be attached.

Reply:
notification manager is lazy loaded, so when no toast is shown yet, it will not be available after disconnect...